files/default/cloudwatch_logs/cloudwatch_log_files.json

```diff
@@ -90,6 +90,25 @@
       ],
       "feature_conditions": []
     },
+    {
+      "timestamp_format_key": "month_first",
```

Comment:
we'll have to check if this format applies to this logfile. It can also be done in a separate PR